service/src/main/kotlin/no/nav/su/se/bakover/service/statistikk/StatistikkServiceImpl.kt

```diff
@@ -44,44 +45,48 @@ internal class StatistikkServiceImpl(
+                        { log.info("Finner ikke person sak med sakid: ${sak.id} i PDL.") },
+                        { aktørId -> publiser(SakStatistikkMapper(clock).map(sak, aktørId)) }
                     )
                 }
+                is Event.Statistikk.SøknadStatistikk.SøknadMottatt ->
+                    publiser(BehandlingStatistikkMapper(clock).map(event.søknad, event.saksnummer, Statistikk.Behandling.SøknadStatus.SØKNAD_MOTTATT))
+                is Event.Statistikk.SøknadStatistikk.SøknadLukket ->
+                    publiser(BehandlingStatistikkMapper(clock).map(event.søknad, event.saksnummer, Statistikk.Behandling.SøknadStatus.SØKNAD_LUKKET))
+                is Event.Statistikk.SøknadsbehandlingStatistikk -> publiser(BehandlingStatistikkMapper(clock).map(event.søknadsbehandling))
+                is Event.Statistikk.RevurderingStatistikk -> publiser(BehandlingStatistikkMapper(clock).map(event.revurdering))
+                is Event.Statistikk.Vedtaksstatistikk -> {
+                    sakRepo.hentSak(event.vedtak.behandling.sakId)!!.let { sak ->
+                        personService.hentAktørIdMedSystembruker(sak.fnr).fold(
+                            ifLeft = { log.error("Finner ikke aktørId for person med sakId: ${sak.id}") },
+                            ifRight = { aktørId ->
+                                val ytelseVirkningstidspunkt = vedtakRepo.hentForSakId(event.vedtak.behandling.sakId)
+                                    .filterIsInstance<VedtakSomKanRevurderes.EndringIYtelse>()
+                                    .minOf { it.periode.fraOgMed }
+
+                                publiser(
+                                    StønadsstatistikkMapper(clock).map(
+                                        event.vedtak,
+                                        aktørId,
+                                        ytelseVirkningstidspunkt,
+                                        sak,
+                                    ),
+                                )
+                            },
+                        )
+                    }
+                }
+                is Event.Statistikk.RevurderingStatistikk.Gjenoppta -> publiser(BehandlingStatistikkMapper(clock).map(event.gjenoppta))
+                is Event.Statistikk.RevurderingStatistikk.Stans -> publiser(BehandlingStatistikkMapper(clock).map(event.stans))
+                is Event.Statistikk.Klagestatistikk -> publiser(BehandlingStatistikkMapper(clock).map(event.klage))
             }
-            is Event.Statistikk.RevurderingStatistikk.Gjenoppta -> publiser(BehandlingStatistikkMapper(clock).map(event.gjenoppta))
-            is Event.Statistikk.RevurderingStatistikk.Stans -> publiser(BehandlingStatistikkMapper(clock).map(event.stans))
-            is Event.Statistikk.Klagestatistikk -> publiser(BehandlingStatistikkMapper(clock).map(event.klage))
+        }.mapLeft {
+            log.error("Feil ved publisering av statistikk", it)
```

Comment:
Dersom StatistikkServiceImpl tar inn en logger med default-verdi i ctor, kan man teste at denne ikke kaster exception for et eller annet tilfelle og sjekke at det logges noe